src/renderer/containers/AssignBeaconNode/assignBeaconNode.scss

```diff
@@ -0,0 +1,10 @@
+.dropdown-container {
```

Comment:
I'm pretty sure once this scss is loaded it will apply the rule to other dropdown containers as well.